vng_api_common/viewsets.py

```diff
@@ -50,9 +52,14 @@ def _check_query_params(self, request) -> None:
         if unknown_params:
             msg = _("Onbekende query parameters: %s" % ", ".join(unknown_params))
             raise ValidationError(
-                {api_settings.NON_FIELD_ERRORS_KEY: msg}, code="unknown-parameters"
+                {api_settings.NON_FIELD_ERRORS_KEY: msg},
+                code=UNKNOWN_PARAMETERS_CODE,
             )
 
     def list(self, request, *args, **kwargs):
         self._check_query_params(request)
         return super().list(request, *args, **kwargs)
+
+    def retrieve(self, request, *args, **kwargs):
```

Comment:
Could you add tests for this? One to check that unknown parameters aren't allowed and one to check that known parameters are allowed